Sources/Commands/Error.swift

```diff
@@ -19,7 +19,7 @@ import var Utility.stderr
 
 public enum Error: Swift.Error {
     case noManifestFound
-    case invalidToolchain
+    case invalidToolchain(problem: String)
```

Comment:
I realize that having a named property here is inconsistent with the other enum values; I will add names to them in a separate diff but wanted to keep this one small.  The reason for adding names will become more clear when we add a solution property as well.